src/api/app/views/webui2/webui/user/_info.html.haml

```diff
@@ -0,0 +1,63 @@
+= webui2_user_image_tag(user, 'home-avatar')
+
+%h4.home-usertitle
+  .home-username
+    = user.realname
+  .home-login
+    = user.login
+
+- unless user.is_nobody?
+  %p
+    = mail_to user.email, title: "Send mail to #{user.name}" do
+      %i.fas.fa-envelope
+      = user.email
+
+- if groups.any?
+  Member of the #{'group'.pluralize(groups.size)}
+  %ul
+    - groups.each do |group|
+      %li= link_to("#{group.title} (#{group.tasks})", group_show_path(group))
+
+- if role_titles.any?
+  Has the #{'role'.pluralize(role_titles.size)}
+  %ul
+    - role_titles.each do |title|
+      %li= title
+
+- if is_user
+  - if user.in_beta?
+    %p
+      .badge.badge-info
+        In public beta program
+    = link_to(user_save_path(user: { login: user.login, in_beta: 0 }), method: :post) do
+      %i.fas.fa-sign-out-alt
+      Leave public beta program
```

Comment:
I have move this links to be just after the information of if the user is in the beta or not